libs/wingsdk/src/target-sim/queue.inflight.ts

```diff
@@ -149,6 +149,14 @@ export class Queue
         if (!fnClient) {
           throw new Error("No function client found");
         }
+
+        // If the function we picked is at capacity, keep the messages in the queue
+        const hasWorkers = await (fnClient as any).hasAvailableWorkers();
```

Comment:
Yep, I just want be careful since I don't really want the code for the sim impl to be bundled / treated as a real dependency of this file. But I think `import type` might do the trick...